.github/workflows/deploy.yml

```diff
@@ -69,15 +69,15 @@ jobs:
       - name: set path
         run: |
           echo "PATH=$(pwd)/node_modules/.bin/:$PATH" >> $GITHUB_ENV
-      - name: unit test & publish coverage
```

Comment:
why did you comment this out?

Reply:
I commented it out because it was failing before. I uncommented it . it looks good now.